community/py-pyrsistent/APKBUILD

```diff
@@ -0,0 +1,53 @@
+# Contributor: Simon Frankenberger <simon-alpine@fraho.eu>
+# Maintainer: Simon Frankenberger <simon-alpine@fraho.eu>
+pkgname=py-pyrsistent
+_pkgname=pyrsistent
+pkgver=0.14.11
+pkgrel=0
+pkgdesc="A number of persistent collections (by some referred to as functional data structures)."
+url="https://github.com/tobgu/pyrsistent/"
+arch="all"
+license="MIT"
+makedepends="python3-dev python2-dev py-setuptools"
+checkdepends="py-hypothesis pytest py-six"
+subpackages="py2-${pkgname#py-}:_py2 py3-${pkgname#py-}:_py3"
+source="https://files.pythonhosted.org/packages/source/${_pkgname:0:1}/$_pkgname/$_pkgname-$pkgver.tar.gz"
+builddir="$srcdir"/$_pkgname-$pkgver
+
+build() {
+	cd "$builddir"
+	python2 setup.py build
+	python3 setup.py build
+}
+
+check() {
+        cd "$builddir"
+        python3 setup.py test
+}
+
+package() {
+	mkdir -p "$pkgdir"
+}
+
+_py2() {
```

Comment:
Do you need python2 support here? Alpine LInux is going to drop python2 support in the near future. docker-compose is using python, so for that it's not needed.

Reply:
You're right, I just added python2 support as the library provides it.
Will remove python2 support and rebase this evening.

Reply:
As this aport is a dependency of py-jsonschema this means that we have to drop python2 support there too. I'm unsure if we should do this now